spec/factories/course_forum_topics.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :forum_topic, class: Course::Forum::Topic.name do
-    forum
+    transient do
+      course { build(:course) }
```

Comment:
I'm not too sure also. I rather keep the factory there first, then figure out how to handle it later on. We can add it to a list of housekeeping issues?